### PR TITLE
Use priority_weight instead of number of task by worker

### PR DESCRIPTION
**Pull request by Valdimus**
For the "Les Vieilles Charrues 2017" we use the number of task by worker to create the dag. This is not optimal at all, so it's better to use the priority_weight. So, now the dag doesn't have to know the number worker.

### CODE REVIEW

**Comment by Valdimus** [comment]
This PR will close #16

File: roles/configuration/templates/campaign.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    'start_date': airflow.utils.dates.days_ago(2),
+    'retries': 0
+}
+
+# Get all the campaigns
+db_client = RestClient("http://OPV_Master:5000")
+campaigns = db_client.make_all(Campaign)
+
+for campaign in campaigns:
+    print(
```
Comment (TomEros) [inline roles/configuration/templates/campaign.py:25]: Why not using logger ?

**Review by TomEros — APPROVED** [review]

**Comment by Valdimus** [comment]
I mad some clean up and I test this PR with campaign "test2" of the Vieille Charrues 2017, all seems to work well ^^.

**Comment by Valdimus** [comment]
Validated by @zyioump

**Pull request merged**